mountpoint-s3/src/main.rs

```diff
@@ -447,7 +447,7 @@ fn main() -> anyhow::Result<()> {
                 let status = receiver.recv_timeout(timeout);
                 match status {
                     Ok('0') => {
-                        println!(
```

Comment:
I could have deleted this line instead but decided to keep it as it could bring more clarity when debugging an issue. Happy to remove if we think that is better.

Reply:
We actually do want to log this explicitly to stdout for positive confirmation to the user when the mount completes.

Instead, we can probably gate this `println!` behind an if condition based on if `--no-log` was provided.

Reply:
Do we really want to do that? I don't think `--no-log` should affect stdout. Unless we are retrofitting it to also mean something like `--quiet`.

Reply:
Good point. Either we make the change or we can close out that bug report.

Reply:
I agree with @passaro that this isn't a `--quiet` option. I will discard this PR and work on closing that bug.